README.md

```diff
@@ -16,10 +16,87 @@ The PDS consists of:
 
 ## Setup
 
-Review the [puppetlabs-puppet\_data\_service](https://github.com/puppetlabs/puppetlabs-puppet_data_service) module for detailed information on how to install, configure, and run the service.
+Here is detailed information to install, configure, and run the service using the [puppetlabs-puppet\_data\_service](https://github.com/puppetlabs/puppetlabs-puppet_data_service) module
+
+The `puppet_data_service` module will install the whole PDS [via its RPM](https://github.com/puppetlabs/puppet-data-service/releases) for you, so you don't have to worry about operationalizing the PDS service itself, dealing with DB setup, migrations, and so on, also it will install the PDS CLI as well.
+
+**Required configuration parameters**
+
+* `puppet_data_service::database_host`
+* `puppet_data_service::pds_token`
+
+**Optional configuration parameters**
+
+* `puppet_data_service::package_source`
+
+### Configure using the PE Console
+
+This setup will help you to quickly configure the PDS in your existing PE server, for advanced Puppet users review the *Configure using roles and Hiera eyaml* section.
+
+1. Add the [puppetlabs-puppet\_data\_service](https://github.com/puppetlabs/puppetlabs-puppet_data_service) module to your control repo
+   - Make sure to add the PDS hiera level in your control-repo's `hiera.yaml`
+2. Configure the two required `roles`
+   - The Database server
+     - Add a new Node Group from the PE Console.
+       ```
+       Parent name: PE Infrastructure
+       Group name: PDS Database
+       Environment: production
+       ```
+     - Add the class `puppet_data_service::database` to the PDS Database group created in the step above
+     - Add (pin) your existing PostgreSQL server `certname` in the Rules tab (it could be the primary server)
+       - In case you want to test the PDS in a different server without PostgreSQL, you can apply the `puppet_enterprise::profile::database` class to your node before following these steps, 
+     - Commit your changes
+   - PDS API server
+     - In the **PE Master** Node group:
+       - Add the new class `puppet_data_service::server`
+       - Include the following parameters:
+         - package_source: The location of the PDS RPM
+         - database_host: The DB server `certname`
+      - In the Configuration data tab:
+         -  Configure the _sensitive_ `pds_token` parameter, this token will be used to create the admin account for the PDS, you can pass a UUID or your own token from your Active Directory/LDAP
```

Comment:
Should we mention for lab purposes you can use something like https://www.uuidgenerator.net/ ?